models/Search/Backend/Data.php

```diff
@@ -510,16 +510,13 @@ public function setDataFromElement(Element\ElementInterface $element)
      */
     protected function cleanupData($data)
     {
-        $data = strip_tags($data);
+        $data = preg_replace('/(<\?.*?(\?>|$)|<.*?>)/s', '', $data);
```

Comment:
this ignores the text if there's a tag after "<" ... e.g. 
"In compliance with UNE 0064-1:2020 . Bacterial filtration efficiency (BFE) ≥ 95%. Breathability (differential pressure) **<60 Pa/cm2. Non-reusable. \<test\>**"

results in "compliance with UNE 0064-1 2020 Bacterial filtration efficiency (BFE) 95%. Breathability (differential pressure)"

Reply:
Changing the regex to this fixes it:
```suggestion
        $data = preg_replace('/(<\?.*?(\?>|$)|<[^<]+>)/s', '', $data);
```

Note: maybe the `+` should be replaced by `*` depending on whether `<>` should be matched or not.